tests/sessions/python_sdk_activity.txtar

```diff
@@ -0,0 +1,32 @@
+oink
+meow
+woof
+moo
+hiss
+
+-- main.py main --
+from random import random
+
+from autokitteh.decorators import noactivity
+from autokitteh.sdktest import *
```

Comment:
Don't `import *` :)
Also, place `sdktest.py` in the txtar file and remove it from the py-sdk code.